### PR TITLE
Form updated for goals 

### PR DESCRIPTION
**Pull request by ashifkhn**
Dropdown has been added and search is implemented with api integration which fetch all the users in the rds server in this pr.

### CODE REVIEW

**Comment by shreya-mishra** [comment]
> Dropdown has been added and search is implemented with api integration which fetch all the users in the rds server in this pr.

please add video

**Comment by ashifkhn** [comment]
Uploading VID_20231110121104.mp4…

**Comment by ashifkhn** [comment]
https://github.com/Real-Dev-Squad/mobile-app/assets/54736284/b1c898bd-01d7-4ca7-99fd-cf8be985adf8

File: src/screens/MemberScreen/MembersPage.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ const MembersPage = () => {
 
   useEffect(() => {
     callMembersApi();
+    console.log("calling members api")
```
Comment (shreya-mishra) [inline src/screens/MemberScreen/MembersPage.tsx]: please remove the log

**Review by shreya-mishra — COMMENTED** [review]
please fix lint issue

**Comment by ashifkhn** [comment]
> please fix lint issue

resolved.

**Review by shreya-mishra — APPROVED** [review]

**Pull request merged**